test_scripts/RC/OnRCStatus/001_notification_by_apps_registration.lua

```diff
@@ -0,0 +1,27 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary:
+-- [SDL_RC] TBD
+--
+-- Description: SDL shall send OnRCStatus notifications to all registered mobile applications and the HMI
```

Comment:
I like the description in PR https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1909/
That have more information on condition/steps/input to the sdl core, and the expected out put